app/models/ongoing_event.rb

```diff
@@ -0,0 +1,43 @@
+class OngoingEvent < ApplicationRecord
+  acts_as_paranoid
+
+  belongs_to :ngo
+  has_many   :participations, dependent: :destroy
+  has_many   :users,          through:   :participations
+
+  validates :title, length: { in: 1..100 }
+  validates :address, presence: true
+  validates :contact_person, presence: true
+
+  scope :newest_first, -> { order(:created_at) }
+  scope :published,    -> { where.not(published_at: nil) }
+
+  def pending?
```

Comment:
I'd rather wait. If we start adding things and it gets ugly, or if there is a third thing, ok. I rarely do this kind of abstraction on only two things.